setup.py

```diff
@@ -19,6 +19,12 @@
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.3',
         'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
```

Comment:
Thanks for updating this.
As we do not test for versions below 3.7 in the CI now, I think we should remove all versions before that one.

Reply:
I was not sure how you wanted that, I removed the older versions.